tests/sentry/tasks/test_post_process.py

```diff
@@ -1280,3 +1282,35 @@ def test_process_transaction_event_clusterer(
         )
 
         assert mock_store_transaction_name.mock_calls == [mock.call(self.project, "foo")]
+
+
+@region_silo_test
+class PostProcessGroupGenericTest(
+    TestCase,
+    SnubaTestCase,
+    OccurrenceTestMixin,
+    CorePostProcessGroupTestMixin,
+    InboxTestMixin,
+    RuleProcessorTestMixin,
+    SnoozeTestMixin,
+):
+    def create_event(self, data, project_id):
```

Comment:
Not a blocker but Gilbert made a test util you can use here: https://github.com/getsentry/sentry/blob/master/tests/sentry/issues/test_utils.py#L64